### ISSUE TITLE
unattended-upgrades not upgrading Node.js

### ISSUE THREAD

**Issue by 7adietri** [nodesource/distributions#48]
Node.js is not automatically being upgraded on my Ubuntu 14.04 machine. The config for unattended-upgrades looks like this:

    Unattended-Upgrade::Allowed-Origins {
        [...]
        "Node Source:trusty";
    };

I noticed that https://deb.nodesource.com/node/dists/trusty/main/binary-amd64/Release doesn't contain an `Archive` property, like http://ppa.launchpad.net/chris-lea/node.js/ubuntu/dists/trusty/main/binary-amd64/Release does. (The config file asks for an `origin:archive` pair.) Could this be the problem?

**Comment by rvagg** [comment]
going to defer to @chrislea on this

**Comment by Amoki** [comment]
I got the same issue when i'm trying to upgrade to node 0.12 :
```bash
curl -sL https://deb.nodesource.com/setup | sudo bash -
sudo apt-get install nodejs
node --version
```
output: 
`v0.10.33`

**Comment by chrislea** [comment]
Hi guys. We're putting 0.12.x in a different repository so that we don't b0rk people who don't want to make the switch yet. We haven't made an announcement, but you can check everything out here:

https://github.com/nodesource/distributions/issues/63#issuecomment-74580073

If you have any issues, please let us know over there. Thanks!

**Comment by retrohacker** [comment]
It appears this was resolved by migrating to the 0.12.x repo.

If this continues to be a problem for you, feel free to reopen and we will help debug!
